CHANGELOG.rdoc

```diff
@@ -1,3 +1,6 @@
+== 2.80.2
```

Comment:
Please set this to unreleased. We should update the Pull Request template to make that clearer.

Reply:
Also, I don't know that this kind of change even merits a Changelog entry.

Reply:
Absolutely.  I wasn't sure what merited a CHANGELOG entry for you guys.  Will change

Reply:
Fixed.  I also verified that many other big name gems use markdown as well for their README, so I don't think there are negative consequences to using this.

If you list out the criteria for what needs a CHANGELOG entry, I can update your PR template in another branch if you'd like.